src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -52,7 +62,8 @@ public void canCheckOutUsingItemAndUserBarcode()
       new CheckOutByBarcodeRequestBuilder()
       .forItem(smallAngryPlanet)
       .to(steve)
-      .at(loanDate));
+      .on(loanDate)
+      .at(UUID.randomUUID()));
```

Comment:
Given that this test is the primary example for how to check out an item using the dedicated API (and to check that it behaves correctly) and that the service point property has been added, please can a check be added that the property is correctly stored on the loan.